web/templates/edited_notice.hbs

```diff
@@ -1,3 +1,4 @@
+<div id="editing_notifications" class="edit_notifications"></div>
```

Comment:
You can't use the same ID for multiple elements in an app. I think the correct answer is the remove the ID.

Also, we use `-`s for separators in all new CSS class names these days.